app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @results = FoodFacade.foods_search(params[:q])
+    @info = FoodFacade.search_info(params[:q])
```

Comment:
Both of these lines are ultimately pinging the same API endpoint. I'll leave some suggestions in other files about how we might be able to ping the API just once and still have both data points available for assignment here.